packages/next/types/index.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="node" />
 /// <reference types="react" />
+/// <reference types="react/experimental" />
```

Comment:
Also pulls in typings for next which include types for [`use`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/36763230a46d1276d7c60464b38cc27daec5e38e/types/react/next.d.ts#L56) and [`cache`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/36763230a46d1276d7c60464b38cc27daec5e38e/types/react/next.d.ts#L83)